tests/tests/fv3_gfs_v16_ca_emis

```diff
@@ -0,0 +1,91 @@
+###############################################################################
+#
+#  FV3 CCPP GFS v16 compiled with 32-bit dynamics test
+#
+###############################################################################
+
+export TEST_DESCR="Compare FV3 32bit CCPP GFS v16 with chemistry with previous version"
+
+export CNTL_DIR=fv3_gfs_v16_ca_emis
+
+export LIST_FILES="atmos_4xdaily.tile1.nc \
+                   atmos_4xdaily.tile2.nc \
+                   atmos_4xdaily.tile3.nc \
+                   atmos_4xdaily.tile4.nc \
+                   atmos_4xdaily.tile5.nc \
+                   atmos_4xdaily.tile6.nc \
+                   phyf000.tile1.nc \
```

Comment:
I believe Jun recently changed phyf/dynf to sfcf/atmf (see the latest code updates for gsl/develop). It would be best to make this switch for newly added tests from the beginning, even though we don't have the update from ufs-weather-model develop yet in the branch.

Reply:
We can also remove the `atmos_4xdaily.tile?.nc` files (and, if we want, turn off the writing of these files). Noboby looks at those when the quilting option is used.

Reply:
> I believe Jun recently changed phyf/dynf to sfcf/atmf

We need to avoid any unnecessary differences between the chemistry test and its non-chemistry version, otherwise comparisons are difficult. All GFS tests in this branch still use phyf and dynf, so the chemistry tests must too, for now.

Reply:
No problem, as long as you change those when you create the PRs for the authoritative repositories.